README.md

```diff
@@ -162,6 +162,10 @@
 
 - [Party Space](https://www.party.space/) - Virtual events platform.
 
+## Experiences
+
+- [Fatal System Error](https://playcanv.as/p/A5qC9ftM/?overlay=false) - An interactive web-based opera experience created by Dumbworld from the Royal Opera House.
```

Comment:
This really belongs in the Advertising section.

Reply:
It's not really an advert for a product though? https://www.roh.org.uk/tickets-and-events/festival/8bit-details

Reply:
Under advertising, I normally include: playable ads and branded experiences. This is kinda the latter.